sufia-models/lib/generators/sufia/models/templates/config/sufia.rb

```diff
@@ -152,6 +152,10 @@
   # NOTE: if you have always sent analytics to GA for downloads and page views leave this commented out
   # config.analytic_start_date = DateTime.new(2014,9,10)
 
+  # Specifies a user account for the Geonames API (please override this value)
+  # Default is empty string
```

Comment:
This is the defaults are set and it's plain to see it's an empty string. I don't think this documentation is necessary.